setup/php/8.1/setup.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+curl -sSLO https://github.com/shivammathur/php-builder/releases/latest/download/install.sh
+chmod a+x ./install.sh
+./install.sh 8.1
+
+# Setup directory structure
+CONFIGURATION_DIRECTORY_PREFIX="/usr/local/php/8.1/etc"
+
+rm -rf /etc/php/8.1
+mkdir -p /etc/php/8.1/{cli,mods-available}
+ln -s $CONFIGURATION_DIRECTORY_PREFIX/php.ini /etc/php/8.1/cli/
+ln -s $CONFIGURATION_DIRECTORY_PREFIX/conf.d /etc/php/8.1/cli/conf.d
+for mod in $(ls /etc/php/8.1/cli/conf.d); do
```

Comment:
Pay attention to the shellcheck output here - will definitely ensure this doesn't blow up on us!